python_search/config.py

```diff
@@ -85,7 +87,7 @@ class ConfigurationLoader:
     Loads the application from the environment.py
     """
 
-    def load(self) -> PythonSearchConfiguration:
+    def load_entries(self) -> PythonSearchConfiguration:
```

Comment:
Could you also remove the return type? it is wrong.

Reply:
alright, done